webpage/language/en.js

```diff
@@ -0,0 +1,251 @@
+let TRANSLATION_EN_US = {
+	"menu": {
+		"title": "Word Clock",
+		"language": {
+			"label": "Language",
+			"de": "🇩🇪 German",
+			"en": "🇺🇸 English",
+			"nl": "🇳🇱 Dutch"
+		}
+	},
+
+	"connection-lost": {
+		"button": "Connect",
+		"text": "The connection to the Word Clock has been lost."
+	},
+
+	"colors": {
+		"h1": "Colors",
+		"with-background": "With Background Color"
+	},
+
+	// Page Functions
+	"functions": {
+		"h1": "Functions",
+		"mode": {
+			"h2": "Mode",
+			"wordclock": "Word Clock",
+			"color": "Color",
+			"seconds": "Seconds",
+			"marquee": "Marquee",
+			"rainbow": "Rainbow",
+			"color-change": "Color Change"
+		},
+		"animation": {
+			"h2": "Animation",
+			"type": {
+				"label": "Type"
+			},
+			"duration": {
+				"label": "Duration",
+				"short": "Short",
+				"medium": "Medium",
+				"long": "Long"
+			},
+			"color": {
+				"label": "Color Animation",
+				"off": "Off",
+				"words": "Colored Words",
+				"letters": "Colored Letters"
+			},
+			"speed": {
+				"label": "Speed"
+			},
+			"demo": {
+				"label": "Demo"
+			}
+		},
+		"settings": {
+			"h2": "Settings",
+			"brightness": "Brightness ",
+			"marquee-text": "Marquee Text",
+			"save": "Save",
+			"speed": "Speed ",
+			"leds": "Number of LEDs ",
+			"position": "Position "
+		}
+	},
+
+	"view": {
+		"h1": "View Options",
+		"front": {
+			"h2": "Front",
+			"text": "Changes to the matrix size are only applied when the clock is restarted.",
+			"variant": "Choice of Clock Variant",
+			"de-10-11": "German 10 × 11",
+			"de-10-11-alt": "German 10 × 11 Alternative",
+			"de-10-11-alt-frame": "German 10 × 11 Alternative Frame",
+			"de-10-11-vertical": "German 10 × 11 Vertical",
+			"de-10-11-clock": "German 10 × 11 Clock",
+			"de-10-11-nero": "German 10 × 11 Nero",
+			"de-11-11": "German 11 × 11",
+			"de-11-11-v2": "German 11 × 11 Version 2",
+			"de-11-11-frame": "German 11 × 11 Frame",
+			"de-22-11-weather": "German 10 × 11 Weather",
+			"de-16-18": "German 16 × 18",
+			"en-10-11": "English 10 × 11",
+			"nl-10-11": "Dutch 10 × 11",
+			"invert-min-dir": "Invert Minute Counting Direction",
+			"mirror-vertically": "Mirror Word Clock Vertically",
+			"mirror-horizontally": "Mirror Word Clock Horizontally"
+		},
+		"language": {
```

Comment:
This section war particularly tricky to translate, so I left it in German for now.